agent/action/arbitrage_result.py

```
@@ -239,7 +243,19 @@ def run(self, context: Context, argv: CustomAction.RunArg) -> bool:
-        
+
+        # 翻页上限:业务可传,但不允许缺省成"无限"。
+        max_scan_pages = _MAX_SCAN_PAGES_DEFAULT
+        if argv.custom_action_param:
+            try:
+                _p = json.loads(argv.custom_action_param)
+                if isinstance(_p, dict) and "max_scan_pages" in _p:
+                    max_scan_pages = max(1, int(_p["max_scan_pages"]))
+            except (ValueError, TypeError) as e:
+                mfaalog.warning(
+                    f"[Arbitrage] ⚠️ custom_action_param 解析失败({e})，翻页上限沿用默认 {max_scan_pages}"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)